src/QmlControls/QGCPipable.qml

```diff
@@ -188,4 +189,37 @@ Item {
             }
         }
     }
+    Rectangle {
+        id:                     buttonScreenShoot
+        anchors.left:           openPIP.right
+        anchors.bottom:         parent.bottom
+        height:                 ScreenTools.defaultFontPixelHeight * 2
+        width:                  ScreenTools.defaultFontPixelHeight * 2
+        radius:                 ScreenTools.defaultFontPixelHeight / 3
+        visible:                true
+        color:                  isDark ? Qt.rgba(0,0,0,0.75) : Qt.rgba(0,0,0,0.5)
```

Comment:
```
color: "black"
opacity: isDark ? 0.75 : 0.5
```

Reply:
done.

Reply:
not really done, I'll update this in a bit.